src/Pages/SolvingQuestion.tsx

```diff
@@ -161,7 +221,7 @@ const FillBtn = styled.button`
 const StrokeBtn = styled.button`
     color: #212121;
     background-color: #fff;
-    border: 1px solid #858585;
+    border: 1px solid #bdbdbd;
```

Comment:
If there is a pre-defined set of colors you are using, I suggest create a seperate file that defines the color values as `const`s.

for example, create utils/colors.ts
```typescript
export const Color = {
  Primary: '#bdbdbd',
}
```

```typescript
border: 1px solid ${Color.Primary};
```

Reply:
Will be much nicer.! After fixing all colors, then I will organize it